src/MinimumNodeService.cpp

```diff
@@ -329,7 +345,9 @@ Processed MinimumNodeService::handleMessage(unsigned int opc, CANFrame *msg)
           else
           {
             // Couldn't find the service.
-            controller->sendGRSP(OPC_RQSD, getServiceID(), CMDERR_INV_PARAM_IDX);
+            controller->sendCMDERR(CMDERR_INV_EN_IDX);
+            // NOTE: error code 9 is really for parameters. But there isn't any better for CMDERR.
+            controller->sendGRSP(OPC_RQSD, getServiceID(), GRSP_INVALID_SERVICE);
```

Comment:
Hmm, not sure what happened here. This is undoing my recent change. Please keep the red code and remove the green code.